data/modules/MissionUtils.lua

```diff
@@ -173,5 +173,59 @@ function MissionUtils.GetNearbyStationPaths(system, range_ly, system_filter, sta
 	return nearby_stations
 end
 
+-- Function: travelTimeLocal
+--
+-- Returns a standardized travel time to a local target
+--
+-- Example:
+--
+-- > travel_time = utils.travelTimeLocal(distance)
+--
+-- Parameters:
+--
+--   distance - the distance to the target in meters
+--
+-- Returns:
+--
+--   the travel time in seconds
+--
+function MissionUtils.travelTimeLocal(distance)
```

Comment:
By convention, we use `PascalCase` for functions defined in a class or namespace like this, with the general idea that a leading capital letter in the module name requires a leading capital letter in the function name. `camelCase` names should be reserved for instance or class variables.

`utils` is a bit of a special case in that it's written in "Lua stdlib style" as it's meant to be a semi-portable extension to the Lua utility functions that isn't directly dependent on any game-specific code.